src/main/java/com/like/minded/backend/service/vendor/VendorServiceImpl.java

```diff
@@ -33,44 +35,89 @@
 @FieldDefaults(level = AccessLevel.PRIVATE, makeFinal = true)
 public class VendorServiceImpl implements VendorService {
 
-    VendorRepository vendorRepository;
+    private final VendorRepository vendorRepository;
+    private final IndoorVendorFactory indoorVendorFactory;
+    private final OutdoorVendorFactory outdoorVendorFactory;
```

Comment:
no need to use private final here. `@FieldDefaults` annotation will take care of it.